drivers/modem/modem_context.h

```diff
@@ -7,6 +7,7 @@
 
```

Comment:
The shell changes should be placed in a separate commit in this PR

Reply:
Done